pkg/messaging/messaging.go

```diff
@@ -29,24 +34,53 @@ them to keep the output clean and focused.
 
 With a free authentication you receive detailed information and solution tips
 for each finding in your console.
-	
+
 All your finding data stays only with us at Code Intelligence
 and will never be shared.`,
 			additionalParams: url.Values{
 				"utm_source":   []string{"cli"},
-				"utm_campaign": []string{"login-message"},
+				"utm_campaign": []string{"run-message"},
+				"utm_term":     []string{"b"},
+			},
+		},
+	},
+	Finding: {
+		{
+			message: `Authenticate with CI Fuzz to get more insights
+on your findings and persist them for a full history.`,
```

Comment:
I think we need to rephrase some of these to CI App, right?
```suggestion
			message: `Authenticate with CI App to get more insights
on your findings and persist them for a full history.`,
```

Reply:
🙈 Good catch!